patient-appointment-management/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "patient-appointment-management",
+  "dependencies": {
+    "twilio": "^3.61.0",
+    "aws-sdk": "^2.925.0",
+    "jszip": "^3.6.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
```

Comment:
The app-specific `package.json` does not need Jest; you can delete the `devDependencies` and `scripts` sections here.

Reply:
Right, so 'npm test' to be run at repo root and not the in the app root. gotcha.